Sources/Rendering/Core/ImageArrayMapper/index.js

```diff
@@ -0,0 +1,171 @@
+import macro from 'vtk.js/Sources/macros';
+import vtkAbstractImageMapper from 'vtk.js/Sources/Rendering/Core/AbstractImageMapper';
+import vtkImageMapper from 'vtk.js/Sources/Rendering/Core/ImageMapper';
+import vtkBoundingBox from 'vtk.js/Sources/Common/DataModel/BoundingBox';
+// import vtkCollection from 'vtk.js/Sources/Common/DataModel/Collection';
+import * as vtkMath from 'vtk.js/Sources/Common/Core/Math';
+import CoincidentTopologyHelper from 'vtk.js/Sources/Rendering/Core/Mapper/CoincidentTopologyHelper';
+
+const { staticOffsetAPI, otherStaticMethods } = CoincidentTopologyHelper;
+const { vtkErrorMacro, vtkWarningMacro } = macro;
+const { SlicingMode } = vtkImageMapper;
+
+// ----------------------------------------------------------------------------
+// vtkImageArrayMapper methods
+// ----------------------------------------------------------------------------
+
+function vtkImageArrayMapper(publicAPI, model) {
+  // Set our className
+  model.classHierarchy.push('vtkImageArrayMapper');
+
+  //------------------
+  // Private functions
+
+  const _computeSliceToSubSliceMap = () => {
+    const inputCollection = publicAPI.getInputData();
+    if (!inputCollection || inputCollection.empty()) {
+      return;
+    }
+
+    const perImageMap = inputCollection.map((image, index) => {
```

Comment:
You can also do `flatMap` here. Up to you if you think a `map` then a `flat` is more readable.

Reply:
I will have to add `flatMap()` to `vtkCollection` first. I think the current form is fine if we want to keep utility functions on `vtkCollection` to a smaller set.